Gemfile.lock

```diff
@@ -561,6 +561,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  arm64-darwin-23
```

Comment:
Silly question - I think I'm also on macOS Sonoma. How come mine doesn't show up/change the list of platforms? 

Or does it go by the first time I used the service and log my machine that way??

Reply:
no idea - it's very persistent on my machine after I do almost anything